api-report/runtime-definitions.api.md

```diff
@@ -246,7 +246,6 @@ export interface IGarbageCollectionData {
 // @public
 export interface IGarbageCollectionDetailsBase {
     gcData?: IGarbageCollectionData;
-    unrefTimestamp?: number;
```

Comment:
Removing this may make the switch to removing setTimeouts harder.

Reply:
This is removed from the old format which isn't used anymore. The unreferencedTimestamp still exists on `IGarbageCollectionState` which is what we use now.